### ISSUE TITLE
[BUG] Broken token in `tags.message` (Risk Message) for 11 rules 

### ISSUE THREAD

**Issue by ccl0utier** [splunk/security_content#3041]
### **Describe the bug**
Some ESCU Rules have broken tokens (missing closing "$") in `tags.message` which results in incorrect Risk Messages in ESCU detections generated from them.

Those rules are: 

title | risk_message
-- | --
ESCU - Disable Logs Using WevtUtil - Rule | WevtUtil.exe used to disable Event Logging on $dest
ESCU - Disable Windows Behavior Monitoring - Rule | Windows Defender real time behavior monitoring disabled on $dest
ESCU - Remote Process Instantiation via DCOM and PowerShell - Rule | A process was started on a remote endpoint from $dest by abusing DCOM using PowerShell.exe
ESCU - Remote Process Instantiation via WMI and PowerShell - Rule | A process was started on a remote endpoint from $dest by abusing WMI using PowerShell.exe
ESCU - Remote Process Instantiation via WinRM and PowerShell - Rule | A process was started on a remote endpoint from $dest by abusing WinRM using PowerShell.exe
ESCU - Remote Process Instantiation via WinRM and Winrs - Rule | A process was started on a remote endpoint from $dest
ESCU - Scheduled Task Creation on Remote Endpoint using At - Rule | A Windows Scheduled Task was created on a remote endpoint from $dest
ESCU - Scheduled Task Initiation on Remote Endpoint - Rule | A Windows Scheduled Task was ran on a remote endpoint from $dest
ESCU - Splunk Command and Scripting Interpreter Risky SPL MLTK - Rule | Abnormally long run time for risk SPL command seen by user $(Search_Activity.user).
ESCU - Windows Service Creation on Remote Endpoint - Rule | A Windows Service was created on a remote endpoint from $dest
ESCU - Windows Service Initiation on Remote Endpoint - Rule | A Windows Service was started on a remote endpoint from $dest

This is from the following SPL:
```
| rest splunk_server=local count=0 /services/saved/searches f=title f=action.correlationsearch.enabled f=action.risk.param._risk_message 
| where 'action.correlationsearch.enabled'=1 
| eval risk_message = 'action.risk.param._risk_message' 
| where isnotnull(risk_message) AND trim(risk_message) != "" AND match(risk_message, "\$") AND NOT match(risk_message, "\$[A-Za-z0-9\_\.]+\$") 
| stats count by title, risk_message 
| fields - count
```

### **Screenshots**

An example from `security_content`:

![image](https://github.com/user-attachments/assets/c3dbadcd-78ae-41d2-9b80-9a5c74356750)

As seen from ESCU detections (Risk Message):

In Enterprise Security:

![image](https://github.com/user-attachments/assets/c7f6470c-a3d9-4334-86a4-1ef32d2412b2)

### **App Version:**
 - ESCU: `4.36.0`

**Comment by patel-bhavin** [comment]
the above PR contains fixes for all the detections except: Splunk Command and Scripting Interpreter Risky SPL MLTK ( we will do this in our Gitlab repo) Thanks for filing this bug and the | rest query!

**Comment by patel-bhavin** [comment]
shipped in 4.37.0! Thank you!
